server/views/pages/create/initialMeetingContact.njk

```diff
@@ -18,7 +18,7 @@
         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full">
-                {% if licence.isEligibleForEarlyRelease %}
+                {% if licence.isEligibleForEarlyRelease and not licence.kind == "HDC" %}
```

Comment:
it'd be nice to get this check pushed into the API at somepoint

Reply:
(I saw the other use of `isEligibleForEarlyRelease` and it hadn't got the extra checks - but I assume that's ok as it's in the `else` so not HDC.., things like that are why it's good to to centralise logic like this)

Reply:
Absolutely, my intention is to make `isEligibleForEarlyRelease` just return `false` for HDC licences, but I need to make sure there's no other places that the flag is used and that that's the behaviour UCD want before I commit to it!